### ISSUE TITLE
make the entire object of class 'remstats'

### ISSUE THREAD

**Issue by SNAnalyst** [TilburgNetworkGroup/remstats#25]
Currently, only the `x$statistics` component of the output of `x <- remstats::remstats()` is of class `remstats`. This is unfortunate, since the `x `object as a whole is supposed to have a particular structure, not just `x$statistics`. 

I would like to create some functions that work on `x` (e.g., subsetting of statistics, info, a print method, a summary method), and that relies on the correct structure for `x`. The same will be true for visualizations of the data in `x`. It would be safer to test whether `x`  has class `remstats` than to only test if `x$statistics` has that class.

In short, let's assign class `remstats` to `x` rather than to `x$statistics`.

**Comment by mlmeijerink** [comment]
The reason that class `remstats` is only assigned to `x$statistics` is that in the (near) future, this will most likely become the only output of the function. If the `remify` and `remstimate` packages are up and running, the other output components become redundant. 

I can assign class `remstats` to `x` rather than to `x$statistics` for now, but the functions that are created for `x` may have to be changed later if the output structure of `remstats` changes. Alternatively, I can make what is now `x$statistics` the only output object and include the other output components as attributes, so that we can still use them for now but this wouldn't affect our code (much) if we remove these components at a later point. What do you prefer?

**Comment by SNAnalyst** [comment]
This is work in progress, so it makes sense that we might have to change code as objects change structure.
That said, all of the components of `x` are useful. Not just as input to `remstimate` and `relevent::rem`, but certainly also for functions that contain descriptives and visualizations. For example, I would like to add activity charts for vertices and for dyads (and some other vis like what is currently available in the `ndtv `and `tsna` packages). This requires the "edgelist" and "riskset". The "statistics" are needed for obvious reasons. 
The component `evls` seems superfluous, although it may be used as input for a function somewhere else that I am not aware of.

Anyway, I would really like to see statistics, edgelist, and riskset maintained inside the object, but don't really care about how the object is then organized, as long as we come up with something that seems to work for now. Then, let's give that larger object class `remstats`. 
That way, I can rewrite the functions that I am now writing for a project and add those to the remverse, making use of these components being part of an overall `remstats` object somehow.

**Comment by mlmeijerink** [comment]
I've updated remstats and have assigned class "remstats" to the larger output object.
